nvflare/fuel/f3/drivers/net_utils.py

```diff
@@ -79,7 +79,8 @@ def get_ssl_context(params: dict, ssl_server: bool) -> Optional[SSLContext]:
 def get_address(params: dict) -> str:
     host = params.get(DriverParams.HOST.value, "0.0.0.0")
     port = params.get(DriverParams.PORT.value, 0)
-
+    if not host:
```

Comment:
Initially I though so too, but it actually is necessary because the value of host is empty string after executing line 80! Somehow the params dict contains the host value as empty str.

Reply:
this is good for this PR, maybe we should fix that how can DriverParams.HOST.value be an empty string `""`